public/localscripts/questionPageScripts.js

```diff
@@ -1,6 +1,6 @@
 function confirmOnUnload() {
     const form = $('form.question-form');
-    const getForm = () => form.serialize();
+    const getForm = () => form.serialize().split('&')[0];
```

Comment:
Wouldn't this affect questions with multiple components? Since (I believe) in that case this would only consider the first field.

Reply:
Would something like this be more appropriate? https://stackoverflow.com/questions/4556172/excluding-certain-inputs-on-serialize

Reply:
Switched to something more robust. Tested/working on my end.

https://github.com/PrairieLearn/PrairieLearn/blob/d56ceaf1bc14e277bfe3757a434e31e9d8a101a5/public/localscripts/questionPageScripts.js#L3